master/internal/api_tasks.go

```diff
@@ -143,35 +144,56 @@ func (a *apiServer) TaskLogs(
 func (a *apiServer) GetTasksCount(
```

Comment:
Could you rename everything from `GetTasks` to `GetActiveTasks`?

(Or maybe this should return both the total task count and the active task count, or allow for filtering to only active tasks. Depends on how it's intended to be used, which I'm not familiar with.)

Reply:
✅